doc/classes/Rect2.xml

```diff
@@ -116,87 +136,100 @@
 			<param index="2" name="right" type="float" />
 			<param index="3" name="bottom" type="float" />
 			<description>
-				Returns a copy of the [Rect2] grown by the specified amount on each side individually.
+				Returns a copy of this rectangle with its [param left], [param top], [param right], and [param bottom] sides extended by the given amounts. Negative values shrink the sides, instead. See also [method grow] and [method grow_side].
 			</description>
 		</method>
 		<method name="grow_side" qualifiers="const">
 			<return type="Rect2" />
 			<param index="0" name="side" type="int" />
 			<param index="1" name="amount" type="float" />
 			<description>
-				Returns a copy of the [Rect2] grown by the specified [param amount] on the specified [enum Side].
+				Returns a copy of this rectangle with its [param side] extended by the given [param amount] (see [enum Side] constants). A negative [param amount] shrinks the rectangle, instead. See also [method grow] and [method grow_individual].
 			</description>
 		</method>
 		<method name="has_area" qualifiers="const">
 			<return type="bool" />
 			<description>
-				Returns [code]true[/code] if the [Rect2] has area, and [code]false[/code] if the [Rect2] is linear, empty, or has a negative [member size]. See also [method get_area].
+				Returns [code]true[/code] if the rectangle has area, or [code]false[/code] if the rectangle is linear, empty, or the [member size] is negative. See also [method get_area].
```

Comment:
How about: "Returns [code]true[/code], if [member size] has positive height and width, or [code]false[/code] otherwise."?